docs/setup.md

```diff
@@ -21,15 +63,11 @@ This project supports development with Lando using the Pantheon recipe. This pro
 3. [Setup local certificate authority](https://docs.devwithlando.io/config/security.html)
 4. Increase Docker resources: Locate the 'Resources' section in your Docker preferences. For most architectures, this project requires at least 3GB of memory and 3 CPUs. Additional CPUs and memory may be helpful but should stay under the halfway mark of your total available resources. Also disable the _'Start Docker when you log in'_ setting under the 'General' tab.
 
-### GitHub
-While the project can be checked out and run locally without it, one must setup an SSH key on their development machine if they wish to push code on any of the yalesites repositories.
-Luckily, [GitHub has an intuitive guide on how to setup an SSH key on your machine and register it with your GitHub Account](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
-
 ### Additional tools
 
-- [Composer](https://getcomposer.org/download/): PHP package manager. Version 2.x.
-- [NVM](https://github.com/nvm-sh/nvm#install--update-script): Recommended for installing and switching between Node versions.
-- Node.js (>=8.0). We recommend installing via NVM.
+- [Composer](https://getcomposer.org/download/): PHP package manager. Version 2.x. (Can use lando instead if you prefer)
+- [NVM](https://github.com/nvm-sh/nvm#install--update-script): Node Version Manager
+- Node.js (>=8.0,<11.0). Via NVM.
 
 ## Project setup script
 
```

Comment:
For this area, I'd also add something like "If you haven't already cloned down the repo..." and then then command to clone down.

Do we need the "alternate manual setup instructions"? I like the idea of explaining what each of the scripts do, including the setup script, but, not sure we want people running the individual commands? Especially if the `setup` script is updated but the docs aren't? I'm not sure about this one so I'd love your input.

Reply:
I agree that the manual isn't technically needed, but was there from before.  I know that Mike V has used the manual before as a way to track down issues that occurred using setup sometimes, so I decided to leave it as is.

I do like the idea of adding something about getting the repo for completeness; that's sort of where I was going toward the top of the document with the quick copy/paste bash commands.

Reply:
I added a small section for cloning the repo for completeness.

Reply:
I agree with @codechefmarc on this one, it is less to maintain and if you really want to run the steps manually, you can break things down by looking at what `npm run setup` is doing.

Reply:
Sorry @myvaughn; if you need them we can look at the script.  :D

I agree, keeping the manual in sync with the script longer term might be problematic.